main.js

```diff
@@ -11,7 +11,7 @@ let debug = process.argv[2] === 'debug';
 
 function createOverlayWindow() {
   const windowOptions = {
-    height: 70,
+    height: 100,
```

Comment:
We should display the error over the icons, we don't need to enlarge the window here

Reply:
Straight on top of them? With a black background or something?

Reply:
Yes, I can do the visual tweaking if you prefer :)

Reply:
As you wish :P